2018/python/generate_2018_error_files.py

```diff
@@ -0,0 +1,9 @@
+#This script will generate a file that will fail a specific edit (though others may also fail)
+#This script relies on the presence of a clean data file
```

Comment:
Can you list which edit types? Is it all S/V/Q?

Reply:
Yes, I just did in the last commit.